### PR TITLE
MOSIP-12228 setting isValid flag to true in case of routing to busAddress

### PR DESCRIPTION
**Pull request by neosoftSatish**

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java [hunk 1]
```diff
@@ -454,7 +454,7 @@ public void testBioDeDupUpdatePacketHandlerProcessingFailure() throws ApisResour
 		Mockito.when(abisHandlerUtil.getUniqueRegIds(any(),any(),any())).thenReturn(matchedRidList);
 
 		MessageDTO messageDto = bioDedupeProcessor.process(dto, stageName);
-		assertFalse(messageDto.getIsValid());
+		assertTrue(messageDto.getIsValid());
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java:457]: Why is this changed to asset true ?

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/test/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStageTest.java [hunk 2]
```diff
@@ -337,21 +343,24 @@ public void testVariousScenarious() throws RegistrationProcessorCheckedException
 		Mockito.when(packetInfoManager.getReferenceIdByRid(Mockito.anyString())).thenReturn(null);
 		stage.deployVerticle();
 		stage.process(dto);
-		assertFalse(dto.getIsValid());
+		assertTrue(dto.getIsValid());
+		assertTrue(dto.getInternalError());
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-abis-middleware-stage/src/test/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStageTest.java:347]: Why all below 4 changes are retriable error cases ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java (hunk 1 above)
Comment (neosoftSatish) [reply registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java:457]: in this case, request is going to manual verification stage. so according to my latest changes isValid should be true...

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/test/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStageTest.java (hunk 2 above)
Comment (neosoftSatish) [reply registration-processor/core-processor/registration-processor-abis-middleware-stage/src/test/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStageTest.java:347]: whenever we catch any exception in middleware stage, it is setting latest Transaction code to REPROCESS, that's why it is retriable...

**Review by vishwa-vyom — APPROVED** [review]
